src/inject/GPMInject/interface/mini.js

```diff
@@ -16,7 +16,7 @@ window.wait(() => {
     Emitter.fire('mini', { state: true });
     oldSize = remote.getCurrentWindow().getSize();
     mainWindow.setSize(MINI_SIZE, MINI_SIZE);
-    mainWindow.setMaximumSize(MINI_SIZE, MINI_SIZE);
```

Comment:
Ah good, I did notice that it held to a square ratio, but I'd assumed it was due to me adding setAspectRatio.

I suppose our opinions of "awful" vary as well. At 350px it still looks darn spiffy to me. ;-) But I can see it not looking as good at, say 500 or 750 or 1000px. But at that point, you're right, it's no longer "mini".

Here's my current flow, with my chat and music on one monitor in a semi-tiled manner:
![workspace 1_014](https://cloud.githubusercontent.com/assets/1148665/13607641/b3541db6-e516-11e5-9e52-4acb2ecd0d49.png)
Previously I had the Google Play Music Extension mini-window in the place where GPMDP is now, and it resized fine to fit in that space.

In consideration, 40 pixels is a really silly thing to debate, so I'm alright with adjusting my screen real-estate accordingly.
![workspace 1_015](https://cloud.githubusercontent.com/assets/1148665/13608514/be3f8586-e51a-11e5-9c94-2842fe37389c.png)
If you find a way to get actual hi-def album art from GPM, then perhaps the restriction can be lifted, but for now, I'll retract my request for expandability and can revert this commit.

The title change though, is much more crucial to my workflow, being able to position or float the window based on it's title, because it does serve quite a different purpose in full mode vs mini mode.